apps/filecopy/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
You might want to add .classpath, .project, .settings/, .metadata/, *.iml, .idea/ as well.